download_current_lead_submission.cwl

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env cwl-runner
+#
+# Authors: Andrew Lamb
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: [python3, /usr/local/bin/download_current_lead_submission.py]
+
+hints:
+  DockerRequirement:
+    dockerPull: quay.io/andrewelamb/python_challenge_utils
```

Comment:
Is this public?  Why quay and not docker hub?  Also I should probably just have an automated build for this.

Reply:
Quay does automated builds.

Reply:
Is there a preference of Quay vs Dockerhub?

Reply:
It sounds like there's no good reason in this case for using Quay over Dockerhub.